### DISCUSSION TITLE
Execution function after modal closing

### DISCUSSION THREAD

**Discussion by axelesp** [mercadopago/sdk-js#32] [Ideas]
Regarding [https://github.com/mercadopago/sdk-js/issues/18] I would like to know or propose that when a user clicks 'close and cancel payment' in the payment modal window, it returns a notification or the possibility of executing a js function to take an action, since it is currently closes and nothing happens on my site anymore, IPN is not triggered.
This when the user has not yet selected a payment method, there is no notification to take an action.

**Comment by Kneehole** [comment]
Hi @axelesp! thanks for your feedback about our sdk. I agree with your proposal, we will take in consideration this functionality! I will add this feature to our future task of adding js callbacks to Checkout Pro!

Thanks!

**Reply by axelesp** [reply]
Great! In the v1 of SDK-js it have a parameter 'onReturn', that assigned to one js function that execute when client close modal checkout, so I could take some action.

**Comment by rafaelfaria** [comment]
@Kneehole Is there any update on this?

**Comment by jjcampis** [comment]
Any updates?
